assets/scss/_variables_project.scss

```diff
@@ -17,3 +17,5 @@ $display4-weight: $_font-weight-light;
 $link-color: #379c9c;
 
 $enable-rounded: false;
+
+$z-index-navbar: 32;
```

Comment:
We're not applying this to the navbar? The navbar could change the actual z-index to 30 and then this value would not be valid anymore. We should ensure the navbar uses this variable to set the z-index.

Reply:
Realised that the change to the navbar is in https://github.com/grpc/grpc.io-docsy/pull/8.

We should define the variable in the docsy theme and used it here instead of the other way around.